app/src/main/java/com/hedvig/app/feature/keygear/ui/itemdetail/KeyGearItemDetailActivity.kt

```diff
@@ -40,13 +41,16 @@ class KeyGearItemDetailActivity : BaseActivity(R.layout.activity_key_gear_item_d
     private lateinit var valuationBinder: ValuationBinder
     private lateinit var coverageBinder: CoverageBinder
     private lateinit var receiptBinder: ReceiptBinder
+    private lateinit var nameBinder: NameBinder
 
     private var isFirstLoad = true
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
         supportPostponeEnterTransition()
+        window.setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_ADJUST_PAN)
```

Comment:
wat dis

Reply:
also, is it something that should be provided by the manifest?

Reply:
Yes i can be moved to the manifest.
It pans the whole activity up when the keyboard is opened, i cant seem to get adjustResize to work so i choose adjustPan instead